TimeCapsuleManagement/forms.py

```diff
@@ -4,9 +4,25 @@
 
 
 class CapsuleForm(forms.ModelForm):
+    VISIBILITY_CHOICES = [
+        (False, 'Private'),  # The stored value is False for "Private"
+        (True, 'Public'),  # The stored value is True for "Public"
+    ]
+    is_public = forms.ChoiceField(
+        choices=VISIBILITY_CHOICES,
+        widget=forms.Select(
+            attrs={'class': 'form-input', 'onfocus': 'adjustLabel(this.id)', 'onblur': 'resetLabel(this.id)'}
+        ),
+        initial=False
+    )
+
     class Meta:
         model = Capsule
         fields = '__all__'
         widgets = {
-            'unsealing_date': DateTimeInput(attrs={'type': 'datetime-local'}),
+            'name': forms.TextInput(attrs={'class': 'form-input'}),
+            'description': forms.Textarea(attrs={'class': 'form-input', 'rows': 6}),
+            'unsealing_date': forms.DateTimeInput(
```

Comment:
Whether the DateTimeInput handle edge cases,
like the user should choose only the date and time which after the current time.

Reply:
fixed